new icon: angular (original, original-wordmark, plain, plain-wordmark)

gabrielfins opened the pull request:
## Double check these details before you open a PR

<!-- Tick the checkboxes to ensure you've done everything correctly -->
- [x] PR does not match another non-stale PR currently opened
- [x] PR name matches the format *new icon: <i>Icon name</i> (<i>versions separated by comma</i>)*. More details [here](https://github.com/devicons/devicon/blob/develop/CONTRIBUTING.md#overview)
- [x] PR's base is the `develop` branch.
- [x] Your icons are inside a folder as seen [here](https://github.com/devicons/devicon/blob/develop/CONTRIBUTING.md#organizational-guidelines)
- [x] SVG matches the standards laid out [here](https://github.com/devicons/devicon/blob/develop/CONTRIBUTING.md#svgStandards)
- [x] A new object is added in the `devicon.json` file as seen [here](https://github.com/devicons/devicon/blob/develop/CONTRIBUTING.md#-updating-the-deviconjson-)

<!-- Refer to the [contributing](https://github.com/devicons/devicon/blob/develop/CONTRIBUTING.md#contributing-to-devicon) guidelines for more details. -->

**This PR closes #991**
<!-- List issues that this PR would close above. Ex: This PR closes #1, #2, #3. -->

## Link to prove your SVG is correct and up-to-date.
<!-- Link to an official page/wiki goes here. Anything that proves your SVGs are the correct ones. -->
https://angular.io/presskit

Panquesito7 reviewed (changes requested):
Hey! Thanks for submitting a PR.
Could you please revert the formatting changes and make another PR for that?

Let us know if you need any help. Thanks. 😄


__
[![sema-logo](https://app.semasoftware.com/img/sema-tray-logo.gif)](https://semasoftware.com/gh) &nbsp;**Summary:** :hammer_and_wrench: This code needs a fix&nbsp; | &nbsp;**Tags:** Not maintainable

kilian-paquier commented on devicon.json:
```diff
@@ -181,6 +181,30 @@
+            ]
+        },
+        "color": "#FFFFFF",
+        "aliases": [
+            {
+                "base": "original",
+                "alias": "plain"
+            }
+        ]
```
Hello @gabrielfins, if you include a specific plain svg, then you don't need to declare an alias between the original and the plain font.

gabrielfins commented:
Is it correct now?

kilian-paquier commented:
Seems good on the `devicon.json ` side.

But I think you need the `angular-plain.svg` to be a svg with only one `<path>`.

Can you confirm @Panquesito7 ?

Snailedlt commented:
@kilian-paquier never heard of that requirement before :O

kilian-paquier commented:
@Snailedlt My bad then ! The `paths are united` in the https://github.com/devicons/devicon/wiki/SVG-Standards seemed like the same for me 😁

Snailedlt commented:
@kilian-paquier Ahh, yes you're right! Hadn't seen that there before :)

Snailedlt reviewed (changes requested):
Some small changes needed, also can we add the wordmark logo too?
https://www.vectorlogo.zone/util/preview.html?image=/logos/angular/angular-ar21.svg
![angular-wordmark](https://www.vectorlogo.zone/logos/angular/angular-ar21.svg)

This is the wordmark logo they use on their website
![image](https://user-images.githubusercontent.com/43886029/177011534-152d5b28-1da1-4530-8991-05d0bebf20fc.png)
Would be nice to have exactly that one, but it's a .png and not a .svg file :/

gabrielfins commented:
Hey y'all, I think all the changes requested are done now. I've added the wordmark logo that @Snailedlt requested and changed the colors to match the one they use in their website. I've also changed the plain version to have only one `<path>` tag and added the "javascript" tag to the icon object in `devicon.json`. If it needs any other changes let me know.

Snailedlt reviewed (approved):
Looks good to me.
Great work! 💯

Panquesito7 reviewed (approved):
Thanks for contributing! 🚀


__
[![sema-logo](https://app.semasoftware.com/img/sema-tray-logo.gif)](https://semasoftware.com/gh) &nbsp;**Summary:** :trophy: This code is awesome&nbsp; | &nbsp;**Tags:** Efficient
